src/api/app/components/bs_request_comment_component.html.haml

```diff
@@ -0,0 +1,19 @@
+.row
+  .col-auto.pr-0
+    = image_tag('icons/chat-icon.svg', height: 18, title: 'Request notification')
```

Comment:
This icon can be replaced with one from Font Awesome: https://fontawesome.com/icons/comment?s=solid

Reply:
As discussed with @danidoni, I pushed changes for this.